lib/preview.ex

```diff
@@ -22,6 +22,21 @@ defmodule Preview do
     |> Stream.run()
   end
 
+  def process_all_latest_versions() do
+    {:ok, names} = Preview.Hex.get_names()
+
+    Task.async_stream(
+      names,
+      fn %{name: name} ->
+        %{"latest_stable_version" => version} = Preview.Hexpm.get_package(name)
```

Comment:
Let's use a modified `Preview.Utils.latest_version?` for this with version data from the repository instead if it's not too much extra work.